src/pages/ModelsIndex/_models.scss

```diff
@@ -22,7 +22,7 @@
       pointer-events: none;
     }
 
-    @media (min-width: $breakpoint-medium) {
+    @media (min-width: 876px) {
```

Comment:
I feel like this needs a comment as to why it's a special value.

Reply:
No problem, done.